Frends.Excel.ConvertToJSON/Frends.Excel.ConvertToJSON/Definitions/DateFormats.cs

```diff
@@ -0,0 +1,21 @@
+﻿namespace Frends.Excel.ConvertToJSON.Definitions;
```

Comment:
Should the namespace be Frends.Excel.ConvertToJSON?

Reply:
In C# best practices namespaces go according to root namespace + folder structure.